objectRecognitionSource/README.md

```diff
@@ -185,6 +187,10 @@ the previous is sent.
     *   (**NOTE:** Previously named "pollRate". For a limited amount of time, both "pollTime" and "pollRate"
     will be valid General Config options, but in the future only "pollTime" will be supported.)
 *   allowQueries: This option allows Queries to be received when set to `true'
+*   maxRunningThreads: Optional. Only used if `pollTime` has been specified. The maximum number of threads running at any 
+given point for polling requests. Must be a positive integer. Default value is 10.
```

Comment:
Is this max threads/queued tasks per source?  per connector?

Reply:
This is the max threads/queued tasks per source. Each source has its own unique thread pool.